Dynatrace/dynatrace-operator
# Add basic support for activeGate auth token

# Description
This change introduces the activeGate auth token support. It basically adds the ability to create an auth token within the operator if the feature flag `feature.dynatrace.com/enable-activegate-authtoken: "true"` is set on the DynaKube.
This is the first PR of the whole activeGate auth token support feature, as token rotation will be included at a later stage.


## How can this be tested?
1. Create an API Token with the correct permission `activeGateTokenManagement.create`
2. Create a Dynakube that also creates an activeGate and add `feature.dynatrace.com/enable-activegate-authtoken: "true"` to it
3. check if the activeGate gets the correct secret mounted and check the logs of the activeGate if it uses the created auth Token


## Checklist
